docs/install_on_kestrel.md

```diff
@@ -0,0 +1,217 @@
+# Installation on Kestrel
+
+This document outlines the process for install hercules onto NREL's kestrel
+computer.  
+
+## Initial Steps
+
+  1. Log into kestrel
+  2. Establish a root folder such as /home/{user-name}/herc_root
+  3. Enter that root folder
+
+
+## Install openfast
+
+(Note this approach mainly follows the instruction set for "CMake with Make for Linux/macOS")
+https://openfast.readthedocs.io/en/dev/source/install/index.html#cmake-with-make-for-linux-macos
+
+
+Then to install the relevent branch:
+
+```
+git clone https://github.com/OpenFAST/OpenFAST.git
+cd OpenFAST
+git checkout tags/v3.4.1
+git switch -c v3.4.1
+mkdir build
+cd build 
+```
+
+Now run the following set of commands:
+
+```
+module purge
+module load craype-x86-spr
+module load intel-oneapi-mkl/2023.2.0-intel
+module load intel-oneapi-mpi/2021.10.0-intel
+module load intel-oneapi-compilers/2023.2.0
+
+export OMP_PROC_BIND=spread
+export KMP_AFFINITY=balanced
+
+cmake .. -DBUILD_SHARED_LIBS=ON -DDOUBLE_PRECISION:BOOL=OFF -DCMAKE_INSTALL_PREFIX:PATH=./install
+
+make -j 36
+make install
+```
+
+
+
+## Install ROSCO
+
+With the same shell which has the above mentioned modules already loaded
+
+(Now following the instructions in the full ROSCO installation)
+https://rosco.readthedocs.io/en/latest/source/install.html#full-rosco-installation
+
+Starting from the ''herc_root'' folder:
+
+```
+git clone https://github.com/NREL/ROSCO.git
+cd ROSCO
+git checkout develop
```

Comment:
Perhaps pick a particular commit hash that works well (a610e5 is the current for develop branch) and stick to it for now to avoid errors.

Reply:
this one I'm not sure but happy to go with the group, @misi9170?  @genevievestarke?